packages/grid/x-data-grid/src/locales/ukUA.ts

```diff
@@ -103,7 +103,7 @@ const ukUAGrid: Partial<GridLocaleText> = {
   // Column menu text
   columnMenuLabel: 'Меню',
   columnMenuShowColumns: 'Показати стовпці',
-  // columnMenuManageColumns: 'Manage columns',
+  columnMenuManageColumns: 'Керування колонками',
```

Comment:
```suggestion
  columnMenuManageColumns: 'Керування стовпцями',
```
Ref: https://learn.microsoft.com/uk-ua/power-apps/maker/data-platform/data-platform-manage-fields